src/main/kotlin/ua/marchenko/artauction/common/annotation/scheduled/annotation/CustomScheduled.kt

```diff
@@ -0,0 +1,5 @@
+package ua.marchenko.artauction.common.annotation.scheduled.annotation
+
+@Retention(AnnotationRetention.RUNTIME)
+@Target(AnnotationTarget.FUNCTION)
+annotation class CustomScheduled(val schedule: String)
```

Comment:
I don't see the usage of your annotations

Reply:
done (added TestController with method with both annotation)